src/content/en/tools/workbox/guides/faq.md

```diff
@@ -0,0 +1,104 @@
+project_path: /web/tools/workbox/_project.yaml
+book_path: /web/tools/workbox/_book.yaml
+description: Frequently Asked Questions when developing with Workbox
+
+{# wf_updated_on: 2018-02-28 #}
+{# wf_published_on: 2018-02-28 #}
+{# wf_blink_components: N/A #}
+
+# FAQ {: .page-title }
```

Comment:
In the interest of fighting "doc creep", where we end up with many separate guides/docs all sitting side by side in the ToC: what's the criteria for adding future information into the "FAQ" guide? We have an existing "Troubleshoot and Debug" guide, for instance—when do we add things to that doc vs. adding it to this FAQ?